docs/how_pool_works.md

```diff
@@ -0,0 +1,198 @@
+This document is an introduction to understanding how the funding pool contract works.
+It doesn't describe all the implementation details or the API,
+it focuses on the basic principles of the core mechanics.
+
+# Overview
+
+The funding pool is a smart contract which creates real-time streams of donations.
+One can start, alter or end the process of sending their funds at any time with immediate effect.
+The flow of funds is automatically maintained and steady over time.
+
+There are 3 roles present in the contract.
+Any Ethereum address can simultaneously take on any of these roles.
+
+- **The sender**: has assets and chooses who do they want to send them to,
+how much, and at what rate
+- **The receiver:** receives funds from senders
+- **The proxy**: receives funds from senders,
+but immediately passes them to receivers of their choice
+
+## The cycles
+
+The whole blockchain history is divided into cycles of equal length.
+Every cycle contains the same, constant number of blocks.
+In the examples below, we assume that the cycle length is 5.
+
+![](how_pool_works_1.png)
+
+# The receiver
+
+There no setup steps for one to become a receiver.
+Any address can receive donations at any time, from any sender.
+The only function of this role is collection of funds sent by others.
```

Comment:
```suggestion
The only function of this role is the collection of funds sent by others.
```

Reply:
I did pass it through Grammarly before submitting the PR and fixed most of the issues. I remember this exact issue and I remember explicitly NOT fixing it. Grammarly treated `collection` as a regular noun, but here it's a noun derived from a verb, which is too abstract to use `the` here.

Reply:
Not sure if it makes a difference but I use a paid version and there are currently 21 correctness alerts left, excluding this one.

Reply:
I've used the free version, which mostly checks the grammar. The premium features are mostly about style, which may or may not be even relevant here. I got 69 issues in total, but with access to details about only 11.

Reply:
Without “the” the sentence sound off to me.